rssfeed/migrations/0001_initial.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Generated by Django 1.9.1 on 2017-01-25 13:04
+# Generated by Django 1.9.1 on 2017-01-26 14:47
```

Comment:
No, there hasn't been a deploy of this code yet, although it would probably be best for me to flatten the migrations?